android/src/main/java/com/rnappauth/RNAppAuthModule.java

```diff
@@ -674,9 +674,15 @@ private void authorizeWithConfiguration(
                 authRequestBuilder.setState(additionalParametersMap.get("state"));
                 additionalParametersMap.remove("state");
             }
+
+            if (additionalParametersMap.containsKey("nonce")) {
+                authRequestBuilder.setNonce(additionalParametersMap.get("nonce"));
+            
```

Comment:
Could we please also add
```java
additionalParametersMap.remove("nonce");
```
here? This will ensure it doesn't also get (redundantly) passed through as a url argument

Reply:
Hey @kadikraman , sure thing !